src/number.ts

```diff
@@ -1,3 +1,38 @@
+/**
+ * Checks if the given number is between two numbers.
+ *
+ * * Starting number is inclusive.
+ * * Ending number is exclusive.
+ * * The start and the end of the range can be ascending OR descending order.
+ * * If end is not specified, it's set to start value. And start is set to 0.
+ *
+ * @param {number} number The number to check.
+ * @param {number} start The start of the range. Inclusive.
+ * @param {number} end The end of the range. Exclusive.
+ * @returns {boolean} Returns `true` if `number` is in the range, else `false`.
+ */
+export const inRange = (
+  number: number,
+  start: number,
+  end?: number
+): boolean => {
```

Comment:
In the interest of great DX I would make this a `function` with two overloads: one with `(number, end)` and one with `(number, start, end)`. Small difference but will give developers a little better typeahead/Intellisense when looking at the arguments.

Reply:
Done 👍🏻